redis/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "redis"
-version = "0.21.5"
+version = "0.21.6"
```

Comment:
This shouldn't be a part of your PR, probably some rebase/merge issue?

Reply:
Got it. Removed.